code/__DEFINES/traits.dm

```diff
@@ -44,6 +44,30 @@
 			}; \
 		} \
 	} while (0)
+#define REMOVE_TRAIT_NOT_FROM(target, trait, sources) \
+	do { \
+		var/list/_L = target.status_traits; \
```

Comment:
Change all the variable names in here. I know the others are like this.

Because macros are non-hygienic, still prefix them with underscores, such as `_status_traits`.

Reply:
you're killing me with this double changes request draft stuff fogle

Reply:
its to display dominance